main/settings.py

```diff
@@ -815,6 +831,16 @@
         "task": "sheets.tasks.process_refund_requests",
         "schedule": crontab(minute=CRON_PROCESS_REFUND_REQUESTS_MINUTES),
     },
+    "generate-course-certificate": {
+        "task": "courses.tasks.generate_course_certificates",
+        "schedule": crontab(
```

Comment:
I'm wondering what your thoughts are on this, I think this would be a good opportunity to make crontab configurations a lot more flexible, what do you think of this PR?

https://github.com/mitodl/ol-django/pull/92

Reply:
I think that will surely make it cleaner and easier for defining the crontab jobs. I can review that PR. 

Although i wonder if we should start migrating to that right from this PR or would it be a good idea to create a PR that migrates all our jobs simultaneously to using the the parser from the ol-django.

Reply:
That's a good point, probably worth doing as a separate PR also because it'll probably require a PR in `saltops` so it'd be best to just have one PR on each repo instead of a few here and there.

Reply:
I agree, In that case, I think you might be able to continue the review on this PR.